### PR TITLE
Embedding logs in Matroska files

### PR DESCRIPTION
**Pull request by aeschweik**
Adding embedded logs alongside the sidecar files for those working with Matroska!

This step currently takes place after the MediaConch policy evaluations because MediaConch parses out the attachments themselves to evaluate them as video files (a reported bug).

### CODE REVIEW

**Comment by privatezero** [comment]
At the risk of cluttering the GUI that much more, maybe this should have an option to turn it off and on? 

That way people who are storing the logs as sidecars wouldn't get hit with double the storage space if they didn't want to? (I know as a percentage of the video file the logs are small, but some of them get pretty big!)

**Comment by retokromer** [comment]
>maybe this should have an option to turn it off and on?

+1

**Comment by kieranjol** [comment]
+1 on this being an option to turn off and on, though i could see myself turning it on when we finally commit to using vrecord. This is really cool work!

File: vrecord [hunk 1]
```diff
@@ -1124,6 +1140,17 @@ else
     done
 fi
 
+if [[ "${embed_choice}" != "${undeclaredoption}" ]] ; then
+    echo ""
```
Comment (dericed) [inline vrecord]: ?

Comment (aeschweik) [reply vrecord]: I was copying the syntax of similar existing options (like framemd5_choice) and didn't stop to think on that line - probably unnecessary/worth changing in all of those?

**Comment by dericed** [comment]
requires 'resolve conflict' please rebase

**Comment by aeschweik** [comment]
Conflicts resolved!

**Pull request merged**